src/components/avatar-builder/Categories.android.ts

```diff
@@ -1,6 +1,8 @@
 import lang from 'i18n-js';
```

Comment:
I just checked and the `Categories.ts` and `Categories.android.ts` files have the same content, it will likely be enough to have just one